modules/token-holders/pages/seed-round.adoc

```diff
@@ -0,0 +1,232 @@
++
+....
+PUBLIC_KEY="$(keysmith public-key -f my-seed-phrase.txt)"
+....
+
+== Create neurons
+
+The next step is to sign and send a message to the genesis token canister (GTC) to create your neuron stake.
+You can do this in one step using a single `dfx canister call` command, or, for added security, as two separate steps that your complete at different times or using different devices.
+For example, you might want to create and sign the message to create neurons on one computer, then copy the message to a different computer or convert the message to a QR code before sending it.
+
+Note the you must have a `dfx.json` file available in the directory from which you are running `dfx` commands that connect to the {IC}. 
+You can create an empty `dfx.json` file in your current directory by running the following command:
+
+....
+touch dfx.json
+....
+
+After creating the `dfx.json` file, you can continue to either the <<Single call>> or <<Separate sign and send steps>> instructions.
+
+=== Single call
+
+To create your neuron stake using a single call:
+
+. Verify that you can access the {IC} by running the following command:
++
+....
+dfx ping ic
+....
+. Call the genesis token canister *canister identifier* and the `create_neurons` method by running a command similar to the following:
++
+....
+dfx canister --network=ic --no-wallet call rno2w-sqaaa-aaaaa-aaacq-cai create_neurons '(\"$PUBLIC_KEY\")'
+....
+
+=== Separate sign and send steps
+
+In some cases, however, you might want to separate the request for creating neurons in separate signing and sending tasks.
+For example, you might want to sign the request but delay sending it for some period of time.
+
+To create your neuron stake using separate sign and send commands:
+
+. Verify that you can access the {IC} by running the following command:
++
+....
+dfx ping ic
+....
+. Create a signed `message.json` file using the principal associated with the identity you created from the PEM file by running a command similar to the following:
++
+....
+dfx canister --network=ic --no-wallet sign --expire-after=1h rno2w-sqaaa-aaaaa-aaacq-cai create_neurons ‘(“PUBLIC_KEY”)’
+....
++
+This command illustrates how to creates a `message.json` to create neurons that is signed using your principal identifier as the message sender and with an expiration window of one hour.
++
+The `--expire-after` option enables you to specify a maximum period of time for the signed message to be valid. The default is 5 minutes (5m).
+If you set an `expire-after` window, however, that the message cannot be sent within the first five minute waiting period. In this example, therefore, you would need to send the message after the initial five minute waiting and within the next 55 minutes for the message to be recognized as valid.
+. Send the signed message to the genesis token canister (GTC) to create a neuron on your behalf by running the following command:
++
+....
+dfx canister --no-wallet --network=ic send message.json
```

Comment:
When send, no need to specified the network because `network` is a field in the message.
Even if the user set `network` on the sending set, it will be ignored. 

`dfx canister --no-wallet send message.json`